src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/PrisonerEntity.kt

```diff
@@ -41,6 +41,9 @@ class PrisonerEntity(
   @Convert(converter = CROIdentifierConverter::class)
   val cro: CROIdentifier? = null,
 
+  @Column(name = "fingerprint")
+  val fingerprint: Boolean? = null,
```

Comment:
A minor point but we do not need to name this if the column is called fingerprint. I appreciate that we have done this for all the other columns though

Reply:
Also can we get away with not making this nullable and defaulting to false? It will be easier to deal with

Reply:
Beat me to it